docs/deploy/prometheus.rst

```diff
@@ -29,3 +29,29 @@ If the ``prometheus-client-nginx`` state file applies to the target, `Node Expor
 .. note::
 
    Bytemark assigns hostnames like ``<server>.<group>.opencontracting.uk0.bigv.io`` to its servers, and implements wildcard DNS for any subdomains. By default, Node Exporter is served from ``prom-client.<hostname>`` on port 80, which works for Bytemark servers without additional configuration. For Hetzner servers, additional configuration is needed. Instead of adding a DNS entry and setting the ``prometheus.client_fqdn`` variable, we simply set the ``prometheus.client_port`` variable and access Node Exporter by the server's IP address.
+
+
+Upgrading Prometheus Software
```

Comment:
We use predicates as headings on how-to pages, so this should be "Upgrade Prometheus".